.github/workflows/ci.yml

```diff
@@ -40,14 +40,22 @@ jobs:
       - name: Bazel tests
         # NOTE: bazel test //... doesn't work (due to node_modules)
         run: bazel test //mesop/...
-      - name: Run playwright test
+      - name: Run playwright test (prod mode)
         run: yarn playwright test
       - uses: actions/upload-artifact@a8a3f3ad30e3422c9c7b888a15615d19a852ae32 # v3.1.3
         if: always()
         with:
           name: playwright-report
           path: playwright-report/
           retention-days: 30
+      - name: Run playwright test (debug/editor mode)
+        run: MESOP_DEBUG_MODE=true yarn playwright test
+      - uses: actions/upload-artifact@a8a3f3ad30e3422c9c7b888a15615d19a852ae32 # v3.1.3
+        if: always()
+        with:
+          name: playwright-report
```

Comment:
Does it matter what name/path that the report gets written to? I'm not familiar with the format.